.circleci/config.yml

```
@@ -182,7 +182,7 @@ jobs:
     # Test building documentation
     docker:
       - image: condaforge/mambaforge
-    resource_class: small
+    resource_class: large
```

```suggestion
    resource_class: medium
```
is probably enough and better for the environment. On my machine it uses about 2GB of RAM, the [medium resource provides 4GB](https://circleci.com/docs/configuration-reference/#x86) so that should be sufficient.

---

you got that right, bud! Working well indeed :beer: